site/content/en/docs/handbook/kubectl.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Kubectl"
+weight: 2
+description: >
+  Use kubectl inside minikube
+aliases:
+  - /docs/kubectl/
+---
+
+By default, kubectl gets configured to access the kubernetes cluster control plane
+inside minikube when the `minikube start` command is executed. 
+
```

Comment:
Maybe add a link to the kubectl documentation here as well ?

https://kubernetes.io/docs/tasks/tools/install-kubectl/

Reply:
I don't think that might be the best place for the link. Would be better to have that in the last of the document. Is that agreeable?

Reply:
Like, `For more help related to kubectl, visit the [docs](https://kubernetes.io/docs/tasks/tools/install-kubectl/).`

Reply:
Got it. Done!